doc/getting-started/readme.md

```diff
@@ -1,10 +1,61 @@
-# GraphQL for Drupal
+#Getting Started
```

Comment:
I think you need spaces in the headers, you can see how its looking here https://github.com/justinlevi/graphql/blob/a89b341f144df0810d1bf598808e4bd71bc6d867/doc/getting-started/readme.md

Reply:
ah its fixed :p awesome. disregard this comment